src/Features/CSharp/Portable/SignatureHelp/Providers/AttributeSignatureHelpProvider.cs

```diff
@@ -57,54 +55,63 @@ private static bool IsArgumentListToken(AttributeSyntax expression, SyntaxToken
-            var accessibleConstructors = attributeType.InstanceConstructors
-                                                      .Where(c => c.IsAccessibleWithin(within))
-                                                      .FilterToVisibleAndBrowsableSymbols(document.ShouldHideAdvancedMembers(), semanticModel.Compilation)
-                                                      .Sort(symbolDisplayService, semanticModel, attribute.SpanStart);
+            var accessibleConstructors = attributeType
+                .InstanceConstructors
+                .Where(c => c.IsAccessibleWithin(within))
+                .FilterToVisibleAndBrowsableSymbols(document.ShouldHideAdvancedMembers(), semanticModel.Compilation)
+                .Sort(symbolDisplayService, semanticModel, attribute.SpanStart);
 
             if (!accessibleConstructors.Any())
             {
-                return null;
+                return;
             }
 
             var anonymousTypeDisplayService = document.Project.LanguageServices.GetService<IAnonymousTypeDisplayService>();
             var documentationCommentFormatter = document.Project.LanguageServices.GetService<IDocumentationCommentFormattingService>();
             var textSpan = SignatureHelpUtilities.GetSignatureHelpSpan(attribute.ArgumentList);
 
             var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
-            return CreateSignatureHelpItems(accessibleConstructors.Select(c =>
-                Convert(c, within, attribute, semanticModel, symbolDisplayService, anonymousTypeDisplayService, documentationCommentFormatter, cancellationToken)).ToList(),
-                textSpan, GetCurrentArgumentState(root, position, syntaxFacts, textSpan, cancellationToken));
+
+            context.AddItems(accessibleConstructors.Select(c =>
+                Convert(c, within, attribute, semanticModel, symbolDisplayService, anonymousTypeDisplayService, documentationCommentFormatter, cancellationToken)));
+
+            context.SetApplicableSpan(textSpan);
+            context.SetState(GetCurrentArgumentState(root, position, syntaxFacts, textSpan, cancellationToken));
         }
 
-        public override SignatureHelpState GetCurrentArgumentState(SyntaxNode root, int position, ISyntaxFactsService syntaxFacts, TextSpan currentSpan, CancellationToken cancellationToken)
+        protected override SignatureHelpState GetCurrentArgumentState(SyntaxNode root, int position, ISyntaxFactsService syntaxFacts, TextSpan currentSpan, CancellationToken cancellationToken)
         {
             AttributeSyntax expression;
-            if (TryGetAttributeExpression(root, position, syntaxFacts, SignatureHelpTriggerReason.InvokeSignatureHelpCommand, cancellationToken, out expression) &&
+            if (TryGetAttributeExpression(root, position, syntaxFacts, SignatureHelpTriggerKind.Other, cancellationToken, out expression) &&
```

Comment:
Elsewhere you used 'Default' as a replacement of InvokeSignatureHelpCommand, here you use Other.  Is that an issue?